src/mscorlib/model.xml

```diff
@@ -683,6 +683,9 @@
       <Member Name="GetHashCode(T)" />
       <Member MemberType="Property" Name="Default" />
     </Type>
+    <Type Name="System.Collections.Generic.EnumEqualityComparer&lt;T&gt;">
```

Comment:
Do we need it also for actual types that inherit from this one? SByteEnumEqualityComparer, ShortEnumEqualityComparer, LongEnumEqualityComparer